docs/feature-set.md

```diff
@@ -0,0 +1,74 @@
+PC Engines apu feature set
+==========================
+
+## Coreboot features
+
+| Feature | Supported |
+|---------|-----------|
+| CBFS option mCPIe2 clk| YES |
+| CBFS option EHCI0 | YES |
+| CBFS option console | YES |
+| CBFS option UARTc/d | YES |
+| Serial number from NIC | YES |
+| Serial number from SPI flash | YES |
+| S1 button | NO`*` |
+| SATA AHCI | YES `**` |
+| Serial console enable/disable | YES`*` |
+| SIMSWAPs | YES`***` |
+| SMBIOS table short partnumber | YES |
+
+`*` - apu5 has no support for S1
+`**` - only mainline releases
+`***` - apu3 seems to have inverted functionality
```

Comment:
@miczyg1  asterisk render incorrectly in one line not in 3. Terminology is confusing e.g. one start is for S1 and serials console, but description say about S1. Please read it through again and fix.